### DISCUSSION TITLE
Select full path when selecting a file

### DISCUSSION THREAD

**Discussion by illud** [wailsapp/wails#1487] [General]
Hello, 
Is there a way to get full path when a file is selected via input, or is there a way to disable path security.
when selecting a file it says C:/fakepath/filename

**Comment by leaanthony** [comment]
No. That's a browser/Webview API limitation.

**Comment by illud** [comment]
I solved it by using [Dialog ](https://wails.io/docs/reference/runtime/dialog) and calling it in frontend

main.go
  ```go
    func (a *App) GameExePath() string {
	    selection, err := runtime.OpenFileDialog(a.ctx, runtime.OpenDialogOptions{
		    Title: "It's your turn!",
	    })
    
	    if err != nil {
		    fmt.Println(err)
	    }
	    return selection
    }
  ```
Frontend
  ```javascript
    import { GameExePath } from "../../wailsjs/go/main/App";
    
    const handleFindFile = async () => {
          await GameExePath().then((result) => {
	          console.log(result)
          });
    }
  ```
Output
  ```sh
    C:/games/monster hunter/mh.exe
  ```

**Reply by leaanthony** [reply]
Nicely done! It would be great to intercept the HTML file button click and do this instead.

**Reply by illud** [reply]
Thanks, is now working. ;P

**Reply by leaanthony** [reply]
On file API?
